nixpkgs-review-checks

```diff
@@ -114,25 +114,26 @@ else
 
       (
         nixpkgs=${NIXPKGS_REVIEW_CHECKS_SOURCE:-$HOME/src/nixpkgs}
-        if [[ -d $nixpkgs ]]; then
-          cd "$nixpkgs" || exit
-        fi
-
         check() {
           type -P "$1"
         }
         review_command="command nixpkgs-review $command $flags"
 
         if check ansi2html && check bc && check bloaty && check coreutils && check curl && check gawk && check gh && check jq && check sed && check hydra-check && check mdcat &&
           check nixpkgs-hammer && check pup && check rg && check savepagenow; then
+          if [[ -d $nixpkgs ]]; then
+            cd "$nixpkgs" || exit
+          fi
           $review_command
         else
           nix_shell=nix-shell
           if type -P cached-nix-shell &>/dev/null; then
             nix_shell=cached-nix-shell
           fi
-
-          $nix_shell -I nixpkgs="$nixpkgs" "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")/shell.nix" --run "$review_command"
+          if [[ ! -d $nixpkgs ]]; then
+            exit
+          fi
+          $nix_shell -I nixpkgs="$nixpkgs" "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")/shell.nix" --run "cd $nixpkgs && $review_command"
```

Comment:
It shouldn't matter in which directory we are because we hardcode the path to the shell here.

Can you try the following and tell me where it points to? You can replace your home directory with `~` if you like.
How are you loading/sourcing?

```suggestion
          echo "${BASH_SOURCE[0]}")")/shell.nix"
          $nix_shell -I nixpkgs="$nixpkgs" "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")/shell.nix" --run "cd $nixpkgs && $review_command"
```

Reply:
Mmh. 
Interestingly, it now correctly outputs the directory of the `shell.nix`. Somehow before, I only got a relative path output (`.`), which then tried to run the `shell.nix` from `nixpkgs` directory..
Strange.

I guess we can close this. Sorry.